### PR TITLE
Initial stab at a blog post for the mac app

### PR DESCRIPTION
**Pull request by orta**

### CODE REVIEW

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "CocoaPods App 1.0"
+author: orta
+categories: cocoapods app release
+---
+
+After about a [year of work](https://github.com/CocoaPods/CocoaPods-app/graphs/contributors), with the release of [CocoaPods 1.0](http://blog.cocoapods.org/CocoaPods-1.0/) the corresponding Mac App is also ready for public usage.
```
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: Missing comma after "CocoaPods 1.0" link.

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "CocoaPods App 1.0"
+author: orta
+categories: cocoapods app release
+---
+
+After about a [year of work](https://github.com/CocoaPods/CocoaPods-app/graphs/contributors), with the release of [CocoaPods 1.0](http://blog.cocoapods.org/CocoaPods-1.0/) the corresponding Mac App is also ready for public usage. 
+
+This is no big reveal, we've been advertising it's existence on our websites for a while. I've already talked a little bit about the tech behind the [Mac App](https://blog.cocoapods.org/CocoaPods-App), so this post is the one that aims to make you say our team should use this instead of the gem.
```
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: "aims to make you say our team should use" <- awkward phrasing.

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+<!-- more -->
+
+## What problem does this solve?
+
+CocoaPods exists within the Ruby ecosystem. The majority of Cocoa developers don't have much overlap with writing ruby programs. Installing and maintaining multiple versions of command-line tools like CocoaPods is something that people have to learn in order to use CocoaPods effectively. 
+
+This creates a problem which [bundler solves](https://guides.cocoapods.org/using/a-gemfile.html) really well, but now you're starting to really dive into the ruby ecosystem and the pre-requisite knowledge to contribute raises again. While I'm on my soap-box, if you're using the cocoapods and fastlane gems in an app, you probably should be using bundler.
+
+So the CocoaPods app hosts everything. Literally, it has it's own self-contained copies of `bzr`, `curl`, `git`, `mercurial`, `ncurses`, `openssl`, `pkg-config`, `ruby`, `scons`, `serf`, `subversion`, `yaml` and `zlib`. This allows the CocoaPods team to ensure consitant behavior across all user setups.
```
Comment (neonichu) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: > consistent

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+This concept is not new for Cocoa developers, as Xcode has been embedding it's own dependencies inside it's `.app` since it moved to the Mac App Store. The advantage from a user's perspective is that it works, and you can install your own crazy version of git, or curl or ruby, and CocoaPods still works.
```
Comment (neonichu) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: > its

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+This is no big reveal, we've been advertising it's existence on our websites for a while. I've already talked a little bit about the tech behind the [Mac App](https://blog.cocoapods.org/CocoaPods-App), so this post is the one that aims to make you say our team should use this instead of the gem.
+
+So let me put [my best hat on](/assets/blog_img/app-1.0/hat-on.jpg), and try to pursuade you that this is a _great idea._
+
+<!-- more -->
+
+## What problem does this solve?
+
+CocoaPods exists within the Ruby ecosystem. The majority of Cocoa developers don't have much overlap with writing ruby programs. Installing and maintaining multiple versions of command-line tools like CocoaPods is something that people have to learn in order to use CocoaPods effectively.
```
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: Ruby is inconsistently capitalized.

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+So let me put [my best hat on](/assets/blog_img/app-1.0/hat-on.jpg), and try to pursuade you that this is a _great idea._
+
+<!-- more -->
+
+## What problem does this solve?
+
+CocoaPods exists within the Ruby ecosystem. The majority of Cocoa developers don't have much overlap with writing ruby programs. Installing and maintaining multiple versions of command-line tools like CocoaPods is something that people have to learn in order to use CocoaPods effectively. 
+
+This creates a problem which [bundler solves](https://guides.cocoapods.org/using/a-gemfile.html) really well, but now you're starting to really dive into the ruby ecosystem and the pre-requisite knowledge to contribute raises again. While I'm on my soap-box, if you're using the cocoapods and fastlane gems in an app, you probably should be using bundler.
```
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: CocoaPods is inconsistently capitalized, maybe link to fastlane?

File: _posts/2016-05-11-CocoaPods-App-1.0.md (hunk 3 above)
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: I'm not sure a reader will know what `bzr`,`curl` etc are. Maybe prefix this with "Literally, it has it's own self-contained copies of everything CocoaPods needs, including..."

"constant"

"behavior"  is Americanized, on purpose?

File: _posts/2016-05-11-CocoaPods-App-1.0.md (hunk 4 above)
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: Missing opportunity to say "it just works" 😛 

`curl` is code-formatted above but not here, and ruby isn't capitalized.

**Comment by orta** [comment]
agreed on all of this so far

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+* Running `pod install` or `pod update`
+* Editing a Podfile
+
+That was basically the spec for the app from day 1. Once [@alloy](https://twitter.com/alloy/) had proved that `pod install/update` was working, [@nate_west](https://twitter.com/nate_west) and I took over and started working on the latter.
+
+### Podfile Editing
+
+Some of these features are mundane, _"hey, wow, syntax highlighting&lt;/sarcasm>"_ but others are really fascinating deep levels of context-specific integration that you'd expect from an IDE.
+
+We've started out with syntax highlighting, and then added useful examples of auto-completion results that show how to use the CocoaPods DSL.
```
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: Should "CocoaPods DSL" be "Podfile DSL"?

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+
+{% breaking_image /assets/blog_img/app-1.0/useful-autocomplete.png, /assets/blog_img/app-1.0/useful-autocomplete.png %}
+
+The app will autocomplete your Pods, including private pods. This feels magical when you don't expect it. We've been having discussions about ways to introduce more of the internal CocoaPods knowledge into the [auto-complete like this](https://github.com/CocoaPods/CocoaPods-app/issues/311) - you're welcome to help out.
```
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: Pods/pods capitalization. autocomplete/auto-complete.

File: _posts/2016-05-11-CocoaPods-App-1.0.md (hunk 7 above)
Comment (orta) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: smart - aye

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+
+The app will generate an overview of what your integration will look like without needing to be installed, so you can understand what CocoaPods _will_ do before it has done it.
+
+{% breaking_image /assets/blog_img/app-1.0/integration-overview.png, /assets/blog_img/app-1.0/integration-overview.png  %}
+
+The app provides a UI for updating your Specs repos, you can find out a little bit more about why this is needed in our [Master spec-repo rate limiting post‑mortem](/Master-Spec-Repo-Rate-Limiting-Post-Mortem).
+
+{% breaking_image /assets/blog_img/app-1.0/update-specs.png, /assets/blog_img/app-1.0/update-specs.png %}
+
+It also has the ability to start a new CocoaPods project, or to even remove CocoaPods from a project entirely. I don't want to give away all of the app's secrets though, so you should give it a test run.
```
Comment (ashfurrow) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: "give it a test run" might link to a download page?

**Comment by ashfurrow** [comment]
Looks good 👍

File: _posts/2016-05-11-CocoaPods-App-1.0.md (hunk 9 above)
Comment (orta) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: there's two big CTAs under it - I'm not worried there

screen shot 2016-05-11 at 11 36 08 am

File: _posts/2016-05-11-CocoaPods-App-1.0.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "CocoaPods App 1.0"
+author: orta
+categories: cocoapods app release
+---
+
+After about a [year of work](https://github.com/CocoaPods/CocoaPods-app/graphs/contributors), with the release of [CocoaPods 1.0](http://blog.cocoapods.org/CocoaPods-1.0/), the corresponding Mac App is also ready for public usage. 
+
+This is no big reveal, we've been advertising it's existence on our websites for a while. I've already talked a little bit about the tech behind the [Mac App](https://blog.cocoapods.org/CocoaPods-App), so this post is the one that makes you say our "team we should use this instead of the gem".
```
Comment (alloy) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: > “our team

instead of

> our “team

?

Comment (orta) [inline _posts/2016-05-11-CocoaPods-App-1.0.md]: *your team - great spot

**Pull request merged**